larray/tests/test_array.py

```diff
@@ -2877,11 +2877,15 @@ def test_to_excel_xlwings(self):
 
     @pytest.mark.skipif(xw is None, reason="xlwings is not available")
     def test_open_excel(self):
+        fpath = abspath('new_excel_file.xlsx')
+        if os.path.exists(fpath):
+            os.remove(fpath)
```

Comment:
new_excel_file was never meant to exist in the "create new file" test, so you should also add this at the end of the test, and/or use different fpath for the new test and/or rename new_excel_file to this_file_does_not_exist.xlsx in the first test, to make it obvious what we mean.